--- xmh-1.0.1/xmh.man.orig	2004-04-23 21:54:58.000000000 +0200
+++ xmh-1.0.1/xmh.man	2006-10-15 23:33:12.736811500 +0200
@@ -1316,8 +1316,8 @@
 An example of \fI.xmhcheck\fP file format, for the folders ``inbox'' and
 ``xpert'':
 .nf
-inbox	/usr/spool/mail/converse
-xpert	/users/converse/maildrops/xpert
+inbox	/var/mail/converse
+xpert	/home/users/converse/maildrops/xpert
 .fi
 .sp
 .SH ACTIONS AND INTERFACE CUSTOMIZATION
