commit dbcedd6db6f6e580d3d3baea1891afb570c10c82
Author: James Cloos <cloos@jhcloos.com>
Date:   Wed Aug 20 10:32:52 2008 -0400

    xaw8 is gone, use xaw7

diff --git a/configure.ac b/configure.ac
index ff6a2ef..5ff3537 100644
--- a/configure.ac
+++ b/configure.ac
@@ -33,7 +33,7 @@ AM_PROG_CC_C_O
 AC_PROG_INSTALL
 
 # Checks for pkg-config packages
-XAW_CHECK_XPRINT_SUPPORT(XMH)
+PKG_CHECK_MODULES(XMH, xaw7)
 AC_SUBST(XMH_CFLAGS)
 AC_SUBST(XMH_LIBS)
 
